lib/core/src/server/manager/manager-webpack.config.ts

```diff
@@ -88,6 +89,7 @@ export default async ({
             RELEASE_NOTES_DATA: JSON.stringify(releaseNotesData),
             DOCS_MODE: docsMode, // global docs mode
             PREVIEW_URL: previewUrl, // global preview URL
+            ...globals,
```

Comment:
Does this open more opportunities for other addons that couldn't set global configuration in their presets? If so, would it be a good idea to document this?